cluster-up/container.sh

```diff
@@ -2,16 +2,24 @@
 
 set -e
 
+if [ -z "$KUBEVIRTCI_PATH" ]; then
+    KUBEVIRTCI_PATH="$(
+        cd "$(dirname "$BASH_SOURCE[0]")/"
+        echo "$(pwd)/"
+    )"
+fi
+
+source ${KUBEVIRTCI_PATH}/hack/common.sh
+
 if [[ $KUBEVIRT_PROVIDER =~ (ocp|okd).* ]]; then
-    POSTFIX="cluster"
-    CONTAINER=$(docker ps | grep kubevirt | grep "${KUBEVIRT_PROVIDER}-${POSTFIX}" | awk '{print $1}')
-    if [ -z $CONTAINER ]; then
-        echo "container was not found"
-        exit 0
-    fi
-    docker exec $CONTAINER bash -c "if ! grep "/root/install/auth/kubeconfig" ~/.bashrc > /dev/null; \
-                                    then echo export KUBECONFIG=/root/install/auth/kubeconfig >> ~/.bashrc; fi"
-    docker exec -it $CONTAINER bash
+    CONTAINER=${provider_prefix}-cluster
+
+    docker exec $CONTAINER bash -c "cp -f /root/install/auth/kubeconfig ~/.kube/config"
```

Comment:
I suppose you need the kubeconfig at that place, but why don't you symlink?

Reply:
Well proper thing would be to link stuff after cluster up somehow.

Reply:
Moved to cluster-up can you recheck ?